contributingGuides/STYLE.md

```diff
@@ -123,9 +144,9 @@ myArray.forEach(item => doSomething(item));
 _.each(myArray, item => doSomething(item));
 
 // Bad
-const myArray = Object.keys(someObject).map(key => doSomething(someObject[key]));
+const myArray = Object.keys(someObject).map(function handleSomething(key) {return doSomething(someObject[key]);});
```

Comment:
It is bad because in the example it talks about using underscore vs Object.keys().map. just added the named function to keep it consistent with the avoiding anonymous function rule

Reply:
Oh got it, thanks for explanation!